FogCreek/Glitch-Community
# Lacy gondola: Private icon

## Links
* https://lacy-gondola.glitch.me/

## Changes:
* Some of this was already changed in previous PRs (we had private icons as badges, emoji, and components IIRC) but now they use Icon when its just the image, and wrapped in our component when interactive (tooltip, toggle)

## How To Test:
* Check out your user page, with private projects and collections holding private projects
* Check out the ~ page of a private project you belong to
* Check out the ~ page of a private project you *don't* belong to 

## Feedback I'm looking for:
Everything look good, work, toggle appropriately?
